code/modules/projectiles/projectile.dm

```diff
@@ -235,9 +235,18 @@
 			//if they have a neck grab on someone, that person gets hit instead
 			var/obj/item/weapon/grab/G = locate() in M
 			if(G && G.state >= GRAB_NECK)
-				visible_message("<span class='danger'>\The [M] uses [G.affecting] as a shield!</span>")
-				if(Bump(G.affecting, forced=1))
-					return //If Bump() returns 0 (keep going) then we continue on to attack M.
+				if(G.affecting.stat == DEAD)
+					var/shield_chance = min(80, (30 * (M.mob_size / 10)))	//Small mobs have a harder time keeping a dead body as a shield than a human-sized one. Unathi would have an easier job, if they are made to be SIZE_LARGE in the future. -Mech

```

Comment:
Uh... Wouldn't the opposite be true, in that a small mob is a less effective shield for a larger mob?

Reply:
I only touched dead mob shielding, in that a small mob, if holding a dead body, is less able to use it as an effective shield since they can't lift it as easily compared to a larger one. If the probability goes through, it's as effective as a shield as it would be if the hostage was alive. I can make shielding itself check the comparative size, though that seemed out of scope for the referenced issue.

Reply:
Yeah, that's fair, write that idea on the Misc projects page, if you can/don't mind?